### PR TITLE
Stuff

### PR DESCRIPTION
**Pull request by hjae78**

### CODE REVIEW

**Comment by hjae78** [comment]
I did not replace any MAX superconductors in recipe as I didn't have time nor could I tell where it was appropriate.

File: src/main/java/gregicadditions/recipes/GARecipeAddition.java [hunk 1]
```diff
@@ -764,8 +764,8 @@ public static void init() {
...
-            CIRCUIT_ASSEMBLER_RECIPES.recipeBuilder().duration(100).EUt(10000).outputs(CRYSTAL_PROCESSOR.getStackForm(4)).inputs(ENGRAVED_CRYSTAL_CHIP.getStackForm(1), SMD_TRANSISTOR_CRYSTAL.getStackForm(8), SMD_CAPACITOR_CRYSTAL.getStackForm(4), MASTER_BOARD.getStackForm(), NANO_CENTRAL_PROCESSING_UNIT.getStackForm(1)).input(wireFine, NiobiumTitanium, 2).fluidInputs(fluidStack).buildAndRegister();
+            CIRCUIT_ASSEMBLER_RECIPES.recipeBuilder().duration(100).EUt(10000).outputs(CRYSTAL_PROCESSOR.getStackForm(4)).inputs(CRYSTAL_CENTRAL_PROCESSING_UNIT.getStackForm(1), SMD_TRANSISTOR_QUANTUM.getStackForm(16), SMD_CAPACITOR_QUANTUM.getStackForm(8), MASTER_BOARD.getStackForm(), NANO_CENTRAL_PROCESSING_UNIT.getStackForm(1)).input(wireFine, NiobiumTitanium, 2).fluidInputs(fluidStack).buildAndRegister();
```
Comment (huneau) [inline src/main/java/gregicadditions/recipes/GARecipeAddition.java:767]: why have you change that ?

File: src/main/java/gregicadditions/recipes/GARecipeAddition.java [hunk 2]
```diff
@@ -1299,6 +1300,8 @@ public static void init2() {
         MIXER_RECIPES.recipeBuilder().duration(400).EUt(480).input(dust, Vanadium).input(dust, Indium, 3).outputs(OreDictUnifier.get(dust, IVSuperconductorBase, 4)).buildAndRegister();
         MIXER_RECIPES.recipeBuilder().duration(2400).EUt(1920).input(dust, Indium, 4).input(dust, Bronze, 8).input(dust, Barium, 2).input(dust, Titanium).fluidInputs(Oxygen.getFluid(14000)).outputs(OreDictUnifier.get(dust, LuVSuperconductorBase, 29)).buildAndRegister();
         MIXER_RECIPES.recipeBuilder().duration(1200).EUt(1920).input(dust, Naquadah, 4).input(dust, Indium, 2).input(dust, Palladium, 6).input(dust, Osmium).outputs(OreDictUnifier.get(dust, ZPMSuperconductorBase, 13)).buildAndRegister();
+        MIXER_RECIPES.recipeBuilder().duration(1200).EUt(1920).input(dust, Naquadah, 4).input(dust, Indium, 2).input(dust, Palladium, 6).input(dust, Osmium).outputs(OreDictUnifier.get(dust, ZPMSuperconductorBase, 13)).buildAndRegister();
```
Comment (huneau) [inline src/main/java/gregicadditions/recipes/GARecipeAddition.java:1303]: you have 2 times ZPMSuperconductorBase recipe

File: src/main/java/gregicadditions/recipes/GARecipeAddition.java (hunk 1 above)
Comment (hjae78) [reply src/main/java/gregicadditions/recipes/GARecipeAddition.java:767]: It is not the finished item, it should still be engraved to become Crystal CPU.

Comment (hjae78) [reply src/main/java/gregicadditions/recipes/GARecipeAddition.java:767]: You can see [here](https://ftbwiki.org/Crystal_Processing_Unit) and [here](https://ftbwiki.org/Crystalprocessor)

**Pull request merged**